modules/ROOT/pages/authentication-authorization/attribute-based-access-control.adoc

```diff
@@ -0,0 +1,134 @@
+:description: How to use Cypher to manage attribute-based access control on a graph.
+
+////
+[source, cypher, role=test-setup]
+----
+CREATE ROLE salesTeam;
+CREATE ROLE engineeringTeamUK;
+CREATE ROLE countryAccessRole;
+----
+////
+
+:page-role: enterprise-edition aura-db-business-critical aura-db-dedicated
+
+[[attribute-based-access-control]]
+= Attribute-based access control
+
+Attribute-based access control, ABAC grants roles based on the evaluation of attributes (or claims) contained in a user's authentication token.
+
+== Setup
+
+Use the following steps to set up attribute-based access control:
+
+1. Enable attribute-based access control in the `neo4j.conf` file by setting the `internal.dbms.feature_flag.attribute_based_access_control` setting to `true`.
+2. Specify which OIDC provider will be used for attribute-based access control by setting the `internal.dbms.security.abac_enabled_authorization_providers` setting to the appropriate providers.
+3. Define the authorization rules that will be used to grant roles based on user attributes by using the Cypher command `CREATE AUTH RULE`, explained below.
+4. Specify which roles will be assigned when the authorization rules are fulfilled using the `GRANT ROLE` command, explained below.
+
+== Create auth rules
+To create an authorization rule, use the following syntax:
+
+[source, cypher25, role="noheader"]
+----
+CREATE AUTH RULE ruleName [IF NOT EXISTS]
+    SET CONDITION conditionExpression
+    [ SET ENABLED {true | false} ]
+----
+
+The conditionExpression is a boolean cypher expression that evaluates user attributes (claims) contained in the authentication token.
+To access an attribute, use the syntax `abac.oidc.user_attribute('<claim_key>')`.
+
+The SET ENABLED clause is optional and can be used to enable or disable the rule upon creation. By default, the rule is enabled.
+
+[NOTE]
+====
+To be able to create auth rules, the user must have the `AUTH RULE MANAGEMENT` privilege.
+====
+
+== Drop auth rules
+To drop an existing authorization rule, use the following syntax:
+
+[source, cypher25, role="noheader"]
+----
+DROP AUTH RULE ruleName [IF EXISTS];
+----
+
+[NOTE]
+====
+To be able to drop auth rules, the user must have the `AUTH RULE MANAGEMENT` privilege.
+====
+
+== Grant roles to auth rules
+To specify which roles will be granted to the user when the authorization rule is fulfilled, use the `GRANT ROLE` command with the following syntax:
+
+[source, cypher25, role="noheader"]
+----
+GRANT ROLE[S] role[, ...] TO AUTH RULE[S] ruleName[, ...];
+----
+
+For information about the `GRANT ROLE` command, see xref:authentication-authorization/manage-roles.adoc#access-control-assign-roles[Assigning roles to users].
+
+[NOTE]
+====
+No roles with associated deny privileges can be assigned using attribute-based access control.
+====
+
+[NOTE]
+====
+To be able to grant roles, the user must have `GRANT ROLE` privilege.
+====
+
```

Comment:
```suggestion

[NOTE]
====
OR semantics are used between roles granted via ABAC and those granted by RBAC mechanisms. For example, given an ABAC rule which grants the `admin` role between the hours of 1900 and 2000, then a user who receives the `admin` role because it appears in their JWT `groups` claim (RBAC) will always have the `admin` role, even outside the hours of 1900 and 2000.  
====

```